src/js/video.js

```diff
@@ -23,15 +25,20 @@ function eventListener(video, ev) {
 		return;
 	}
 
+	fireEvent(ev.type, video, {
+		progress: video.getProgress(),
+		duration: video.getDuration()
+	});
+}
+
+function fireEvent(action, video, extraDetail = {}) {
 	const event = new CustomEvent('oTracking.event', {
-		detail: {
-			action: ev.type,
-			advertising: video.opts.advertising,
+		detail: Object.assign({
 			category: 'video',
+			action,
+			advertising: video.opts.advertising,
 			contentId: video.opts.id,
-			progress: video.getProgress(),
```

Comment:
They haven't been removed from the existing events (all existing events behave the same), they're just not added to the new event this PR adds